docs/range-tokens/summary.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Range Tokens Summary
+sidebar_label: Summary
+---
+
+The range token enables a DAO or nascent project to use its native token as collateral to borrow funds. At maturity, if the debt is not paid, the range token holder is instead compensated with an equivalent amount of the collateral (the native token) using the settlement price of the native token to determine the number of tokens. 
+
+A range token can be viewed similarly to convertible debt. In the venture capital world, convertible debt allows startup companies to receive funding today without issuing equity upfront. The range token holder is effectively short a put option and will have exposure to the downside of the native token below a certain price. To compensate the range token holder for taking on this “default” risk, the holder is rewarded a call option on the native token. A minimum number of native tokens is given to the range token holder no matter how much the native token rallies. The combination of the structure creates a tradeoff between the seller and the buyer.
+
+![](/docs/range-tokens/range_token_formula.png)
+
+## Range Token Example
+
+DAOs have become massive with some treasuries valued in the billions of dollars. However, the majority of these assets are in the native token of the project with only a fraction in liquid assets or stable coins. An example of a project using range tokens would be if the Uniswap treasury decided to use $UNI tokens as collateral to borrow USDC. The treasury could create the following terms for the range token contract:
```

Comment:
```suggestion
DAOs have become massive with some treasuries valued in the billions of dollars. However, the majority of these assets are in the native token of the project with only a fraction in liquid assets or stable coins. An example of a project using range tokens would be if the UMA treasury decided to use $UMA tokens as collateral to borrow USDC. The treasury could create the following terms for the range token contract:
```

I know this is all grandfathered in from the previous tutorial, but can we replace all of the UNI examples with UMA?

Reply:
Updated all UNI examples to UMA.